anta/logger.py

```diff
@@ -99,7 +99,7 @@ def anta_log_exception(exception: BaseException, message: str | None = None, cal
 
     Args:
     ----
-        exception: The Exception being logged
+        exception: The Exception being logged.
         message: An optional message
```

Comment:
C O N S I S T E N C Y
```suggestion
        message: An optional message.
```